src/components/OverviewCard.jsx

```diff
@@ -74,7 +75,7 @@ const OverviewCard = ({
       } = frontMatter;
 
       let collectionPageData
-      if (!isResource && chosenCategory) {
+      if (!isResource && !isNewCollection) {
         // User selected an existing page collection
         const { collectionPages } = await retrieveThirdNavOptions(siteName, chosenCategory, true)
         collectionPageData = collectionPages
```

Comment:
Currently, this fails to account for the case when we attempt to move a page from a collection back to unlinked page

Reply:
Good catch, resolved in https://github.com/isomerpages/isomercms-frontend/pull/294/commits/79bb5f447355ece52ed299aed6e8578a2bf94378